generated/nidaqmx/stream_readers.py

```diff
@@ -247,6 +247,44 @@ def read_one_sample(self, timeout=10):
     def read_waveform(
         self, number_of_samples_per_channel=READ_ALL_AVAILABLE, timeout=10
     ) -> AnalogWaveform[numpy.double]:
+        """
+        Reads a waveform from a single analog input channel in a task.
+
+        Args:
+            number_of_samples_per_channel (Optional[int]): Specifies the
+                number of samples to read.
+
+                If you set this input to nidaqmx.constants.
+                READ_ALL_AVAILABLE, NI-DAQmx determines how many samples
+                to read based on if the task acquires samples
+                continuously or acquires a finite number of samples.
+
+                If the task acquires samples continuously and you set
+                this input to nidaqmx.constants.READ_ALL_AVAILABLE, this
+                method reads all the samples currently available in the
+                buffer.
+
+                If the task acquires a finite number of samples and you
+                set this input to nidaqmx.constants.READ_ALL_AVAILABLE,
+                the method waits for the task to acquire all requested
+                samples, then reads those samples. If you set the
+                "read_all_avail_samp" property to True, the method reads
```

Comment:
> add documentation (generated)

Generated from what? Is this copied from another method or do I need to compare it more closely?